File: client-js/src/controller/api/dto/TestResultsDto.ts
import TargetInfoDto from "./TargetInfoDto"
import AdditionalInfoDto from "./AdditionalInfoDto";
import ExtraHeuristicsDto from "./ExtraHeuristicsDto";


export default class TestResultsDto {

    targets: Array<TargetInfoDto> = [];


    additionalInfoList = new Array<AdditionalInfoDto>();

    extraHeuristics = new Array<ExtraHeuristicsDto>();
}

File: outdated/outdated-client-dotnet/src/EvoMaster.Controller.Api/dto/TestResultsDto.cs
using System.Collections.Generic;

namespace EvoMaster.Controller.Api {
    public class TestResultsDto {
        public IList<TargetInfoDto> Targets { get; set; } = new List<TargetInfoDto>();

        /**
     * This list is sorted based on the action indices
     */
        public IList<AdditionalInfoDto> AdditionalInfoList { get; set; } = new List<AdditionalInfoDto>();

        public IList<ExtraHeuristicsDto> ExtraHeuristics { get; set; } = new List<ExtraHeuristicsDto>();
    }
}